include/sdf/JointAxis.hh

```diff
@@ -31,11 +31,34 @@ namespace sdf
 {
   // Inline bracket to help doxygen filtering.
   inline namespace SDF_VERSION_NAMESPACE {
-  //
 
   // Forward declare private data class.
   struct PoseRelativeToGraph;
   template <typename T> class ScopedGraph;
+  /// \brief Helper class to hold contents of the mimic joint tag.
+  /// The parent and child joint positions are related with:
+  /// multiplier * (parentJoint - reference) + offset = childJoint.
+  class SDFORMAT_VISIBLE mimicJoint
```

Comment:
nit: class names should start with uppercase letters.

Reply:
Done https://github.com/gazebosim/sdformat/pull/1166/commits/e3813f68a3ef9ba9f7f845a6bd21a06e89b4f69a